SPOOP/Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour {
+
+	public float speed;
+	public float height;
+	public float runSpeed;
+
+	private Rigidbody rb;
+	private float runMove;
+
+	void Start ()
+	{
+		rb = GetComponent<Rigidbody>();
+		runMove = speed * runSpeed;
+	}
+
+	void FixedUpdate ()
+	{
+		// regular player movement
+		float moveHorizontal = Input.GetAxis ("Horizontal");
+		float moveVertical = Input.GetAxis ("Vertical");
+
+		Vector3 movement = new Vector3 (moveHorizontal, 0.0f, moveVertical);
+
+		// check if running
+		if (Input.GetKey (KeyCode.LeftShift)) {
+			rb.AddForce (movement * runMove);
+		} else {
+			// not running
+			rb.AddForce (movement * speed);
+		}
+
+		// jumping
+		//TODO: check if player is already jumping

```

Comment:
maybe make a "floorHeight" variable that this checks for. default to zero and then when we go to different floors it can be updated accordingly